docs/config/README.md

```diff
@@ -378,6 +379,30 @@ a single character. For `fish_style_pwd_dir_length = 2`, it would be `/bu/th/ci/
 truncation_length = 8
 ```
 
+## Docker Context
+
+The `docker_context` module shows the currently active
+[Docker context](https://docs.docker.com/engine/context/working-with-contexts/) if it's not set to
+`default`.
+
+### Options
+
+| Variable                | Default       | Description                                                                  |
+| ----------------------- | ------------- | ---------------------------------------------------------------------------- |
+| `symbol`                | `"🐳 "`       | The symbol used before displaying the Docker context .                       |
+| `style`                 | `"bold blue"` | The style for the module.                                                    |
+| `only_with_compose_yml` | `false`       | Only show when there's a `docker-compose.yml` file in the current directory. |
+| `disabled`              | `true`        | Disables the `docker_context` module.                                        |
```

Comment:
We generally put the `style` and `disabled` variables at the bottom of the table.
```suggestion
| `only_with_compose_yml` | `false`       | Only show when there's a `docker-compose.yml` file in the current directory. |
| `style`                 | `"bold blue"` | The style for the module.                                                    |
| `disabled`              | `true`        | Disables the `docker_context` module.                                        |
```